documentation/src/main/asciidoc/repositories/Configuration.adoc

```diff
@@ -60,7 +60,7 @@ annotationProcessor 'org.hibernate.orm:hibernate-processor:7.0.0'
 
 === Excluding classes from processing
 
-There's three ways to limit the annotation processor to certain classes:
+There are three ways to limit the annotation processor to certain classes:
```

Comment:
Note that I'm letting you change these because in principle (to a linguistic prescriptivist) "there's three ways" is grammatically "incorrect" in written English. But note that "there's many things" is something that's _incredibly_ common in informal English and this phrase sounds completely correct to a native speaker. I would never ever notice this "mistake" in either written or spoken English.

Reply:
reverted back

Reply:
hehe, then I reverted back the usage of `there are ...`. Also, the changes align with the other two docs (Hiberbate 6 introduction and Hibernate Query Language) I'd made similar changes.